source/index.html

```diff
@@ -170,6 +170,15 @@
 			color: #fff;
 			background: #24CDE8;
 		}
+		
+	           h1 {
+			font-weight: 300;
+			font-size: 40px;
+			line-height: 78px;
+			text-transform: uppercase;
+			color: #404040;
+                         }  
```

Comment:
Some of these lines start with spaces instead of tabs.  I believe all of the files in this project use tabs for indentation, so we should follow that consistently.  Most editors have a setting that will automate this, my guess is that the editor you are using is defaulting to spaces when you press the tab key for some reason.

Reply:
Note: The same problem occurs throughout this document.  Easiest way to fix it is probably to enable whitespace rendering in your editor (how you do this varies by editor) and look for spaces at the beginning of any lines.

Reply:
Thank you. I haven't used an editor when I edited the page before (I only used notepad) :)